testnets/mantradukongtestnet/assetlist.json

```diff
@@ -0,0 +1,68 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "mantradukongtestnet",
```

Comment:
the testnet chain_name must be "mantrachaintestnet"

Reply:
Thanks for the review @JeremyParish69 
What would you recommend for the name as `mantrachaintestnet` is already taken by the first testnet of Mantra (Hongbai)

cf: https://github.com/cosmos/chain-registry/blob/d7165465d3758d9af3f846e4f68c765dd946e45e/testnets/mantrachaintestnet/chain.json#L3

Dukong is the second one

Reply:
My apologies, I assumed it wasn't already taken. In that case, the name should be `mantrachaintestnet2`